buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -389,7 +389,7 @@ class BeamModulePlugin implements Plugin<Project> {
         avro_tests                                  : "org.apache.avro:avro:1.8.2:tests",
         bigdataoss_gcsio                            : "com.google.cloud.bigdataoss:gcsio:$google_cloud_bigdataoss_version",
         bigdataoss_util                             : "com.google.cloud.bigdataoss:util:$google_cloud_bigdataoss_version",
-        bigtable_client_core                        : "com.google.cloud.bigtable:bigtable-client-core:$bigtable_version",
+        bigtable_client_core                        : "com.google.cloud.bigtable:bigtable-client-core:1.8.0",
```

Comment:
Any particular reason this moves away from `$bigtable_version`?

Reply:
It's only used here, and so it doesn't need to be kept in sync with other packages. I can certainly change it back, though.

Reply:
No problem. If it's only used once, the variable indirection only makes it harder to read. Thanks for cleaning it up.